encrypted-media-respec.html

```diff
@@ -1542,17 +1542,27 @@
                     <li><p>Let <var>session id</var> be a unique <a def-id="session-id"></a> string.</p>
                       <p>If the result of running the <a def-id="is-persistent-session-type-algorithm"></a> algorithm on <var>session type</var> is <code>true</code>, the ID MUST be unique within the <a def-id="origin"></a> of this object's <a def-id="document-concept"></a> over time, including across Documents and browsing sessions.</p>
                     </li>
-                    <li><p>Let <var>message</var> be a license request for the <var>requested license type</var> generated based on the <var>sanitized init data</var>, which is interpreted per <var>initDataType</var>.</p>
+                    <li><p>Let <var>message</var> be a license request for the <var>requested license type</var> generated based on the <var>sanitized init data</var> interpreted per <var>initDataType</var>, or an individualization request.</p>
                       <p>The <var>cdm</var> MUST NOT use any stream-specific data, including <a def-id="media-data"></a>, not provided via the <var>sanitized init data</var>.</p>
                       <p>The <var>cdm</var> SHOULD NOT store session data, including the session ID, at this point. See <a def-id="session-storage"></a>.</p>
+                      <o>If <var>message</var> is an individuallization request, then in a subsequent call to <a def-id="update"></a> the CDM MUST generate a license request for the <var>requested license type</var>  based on the <var>sanitized init data</var>, which is interpreted per <var>initDataType</var>.</p>
+                      <p class="issue">
+                        <a href="https://github.com/w3c/encrypted-media/issues/31">Issue 31</a> proposes that the above requirement be formally
```

Comment:
I filed #207 to specifically track future work. We can use that number here and close #31 with this PR.